atomic

```diff
@@ -273,6 +280,9 @@ if __name__ == '__main__':
     imagesp.set_defaults(func='images')
     imagesp.add_argument("--prune", action="store_true",
                          help=_("prune dangling images"))
+    imagesp.add_argument("--oci", dest="oci",
+                         action='store_true', default=False,
+                         help=_('Show OCI containers'))
 
```

Comment:
Wouldn't the default show all container images?  Then we can add a selection criteria to limit the type of images.

--filter oci,docker,rkt?

What is an OCI IMage? Seems a lot like a docker image?